src/mono/mono/mini/mini-codegen.c

```diff
@@ -2166,7 +2166,7 @@ mono_local_regalloc (MonoCompile *cfg, MonoBasicBlock *bb)
 MONO_RESTORE_WARNING
 
 CompRelation
-mono_opcode_to_cond (int opcode)
+mono_try_opcode_to_cond (int opcode)
```

Comment:
The method names is a little confusing to me. I feel these ones are better:
`mono_try_opcode_to_cond` -> 'mono_opcode_to_cond'
`mono_opcode_to_cond` -> `mono_opcode_to_cond_with check`

Reply:
Renamed the try version to unchecked, didn't want to rename all callers of the old one.